### PR TITLE
LPS-20304

### PR DESCRIPTION
**Pull request by MohammedElkhaldi**

### CODE REVIEW

**Comment by Youness-ELMO** [comment]
ci:test:sf

**Comment by AkramYoussoufi** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/myposhitests/FormTests.testcase [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		task ("And The field titles are translated to french") {
+			Form.addTranslation(localizedLanguage = "French (France)");
+
+			TestForm.editFieldsForm(
+				fieldValue_french1 = "Nom Prenom",
+				fieldValue_french2 = "Age",
+				formDescription = "form en frc",
+				formName = "form frc",
+				formPageName = "Creation formulaire en frc",
+				formTitleNumber = 1);
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/myposhitests/FormTests.testcase:48]: This looks like a copy/paste from adnan...

File: portal-web/test/functional/com/liferay/portalweb/tests/myposhitests/MyBlogTests.testcase [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Can Create a Blog"
+	@priority = 5
+	test CanCreateBlog {
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/myposhitests/MyBlogTests.testcase:36]: this name is not good, talk with your teammates to know, about the problem ...

File: portal-web/test/functional/com/liferay/portalweb/tests/myposhitests/MyBlogTests.testcase [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+			AssertElementPresent(
+				key_entryTitle = "Extra Blog",
+				key_tagName = "Extra Blog",
+				locator1 = "BlogsEntry#TAG");
+		}
+	}
+
+	@description = "Can Delete a Blog"
+	@priority = 5
+	test CanDeleteBlog {
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/myposhitests/MyBlogTests.testcase:57]: this name is not good, talk with your teammates to know, about the problem ...

File: portal-web/test/functional/com/liferay/portalweb/tests/myposhitests/MyBlogTests.testcase [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+			AssertElementNotPresent(
+				key_entryTitle = "Blog 2",
+				key_tagName = "Blog 2",
+				locator1 = "BlogsEntry#TAG");
+		}
+	}
+
+	@description = "Can Edit a Blog"
+	@priority = 5
+	test CanEditBlog {
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/myposhitests/MyBlogTests.testcase:78]: this name is not good, talk with your teammates to know, about the problem ...

File: portal-web/test/functional/com/liferay/portalweb/tests/myposhitests/TestForm.macro [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+			fieldName = ${fieldName_french1},
+			fieldValue = ${fieldValue_french1});
+
+		FormFields.editText(
+			fieldName = ${fieldName_french2},
+			fieldValue = ${fieldValue_french2});
+
+		FormPortlet.submit();
+
+		Form.submitAgain();
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/myposhitests/TestForm.macro:102]: You can create 1 method to use both situations ...

**Review by manoelcyreno — CHANGES_REQUESTED** [review]

**Pull request closed without merging**